manipulation/src/functTest/resources/kafka/build.gradle

```diff
@@ -622,10 +622,11 @@ task jacocoRootReport(type: org.gradle.testing.jacoco.tasks.JacocoReport) {
   description = 'Generates an aggregate report from all subprojects'
   dependsOn(javaProjects.test)
 
-  additionalSourceDirs.from = javaProjects.sourceSets.main.allSource.srcDirs
-  sourceDirectories.from = javaProjects.sourceSets.main.allSource.srcDirs
-  classDirectories.from = javaProjects.sourceSets.main.output
-  executionData.from = javaProjects.jacocoTestReport.executionData
+  // XXX: Commented out due to incompatibility with Gradle 4.10
+  //additionalSourceDirs.from = javaProjects.sourceSets.main.allSource.srcDirs
+  //sourceDirectories.from = javaProjects.sourceSets.main.allSource.srcDirs
+  //classDirectories.from = javaProjects.sourceSets.main.output
+  //executionData.from = javaProjects.jacocoTestReport.executionData
```

Comment:
Not sure why these don't work; tests seem to pass without them.

Reply:
I think that test was introduced for NCLSUP175 (Handle archivesBaseName for subprojects) so removing those should be fine.